src/components/Modals/CourseMenu.vue

```diff
@@ -1,6 +1,16 @@
 <template>
   <div class="courseMenu">
     <div class="courseMenu-content">
+      <div class="courseMenu-section" @click="openNoteModal(courseObj.code)">
+        <div class="courseMenu-left">
+          <img
+            class="courseMenu-icon"
+            src="@/assets/images/noteIconSmall.svg"
```

Comment:
I know we might change the image, but I believe the icon is a bit bigger than the other icons so it looks unaligned

Reply:
Yes that is the case, we'd have to rearrange all the icons sidebar style to be centered not right-aligned and have a grid or dynamic offset calculation otherwise — I'd just find a new icon honestly.